apps/vscode/extension/src/unfurl.ts

```diff
@@ -1,4 +1,4 @@
-import cheerio from 'cheerio'
+import { load } from 'cheerio'
```

Comment:
moved this over for use in the vscode extension and it seemed to have different eslint rules so I had to a make a couple of minor syntax edits